sites/site-utilities/statics/server/server.js

```diff
@@ -1,10 +1,14 @@
-var path = require("path");
 var express = require("express");
 var fallback = require("express-history-api-fallback");
+var helmet = require("helmet");
```

Comment:
Does this need to be enumerated as part of the package.json? I don't see this dependency anywhere in the repo currently.

Reply:
Good catch, because NPM is used to install from this package.json file on each web server.